Formula/dcmtk.rb

```diff
@@ -29,13 +29,15 @@ class Dcmtk < Formula
   uses_from_macos "libxml2"
 
   def install
-    system "cmake", "-S", ".", "-B", "build/shared", *std_cmake_args,
+    args = std_cmake_args + ["-DDCMTK_WITH_ICU=OFF"]
```

Comment:
Giving this a try. Debian builds with this option (https://salsa.debian.org/med-team/dcmtk/-/blob/master/debian/rules#L38).